doc/refman/RefMan-pre.tex

```diff
@@ -1335,6 +1335,157 @@ \section*{Credits: version 8.7}
+  for CIC, by Matthieu Sozeau. Handling of evars in the VM (the kernel
+  still does not accept evars) and bugfix in the handling of universes
+  in the VM (backported to 8.7 as well) by Maxime Dénès.
+\item Gallina: always use the projection printing style for primitive
+  projections by Maxime Dénès.
+\item Notations: many improvements on recursive notations and
+  integration with pattern binding by Hugo Herbelin.
+\item Proof language: tacticals for profiling, timing and checking
+  success or failure of tactics by Jason Gross. The focusing bracket
+  \texttt{\{} supports single-numbered goal selectors,
+  e.g. \texttt{2:\{}, by Théo Zimmermann.
+\item Vernacular: deprecation of commands and more uniform handling of
+  the `Local` flag, by Vincent Laporte and Maxime Dénès, part of a
+  larger attribute system overhaul. Experimental ``Show Extraction''
+  command by Pierre Letouzey. Coercion now accepts ``Prop'' or ``Type''
+  as a source by Arthur Charguéraud. ``Export'' modifier for options
+  allowing to export the option to modules that ``Import'' and not only
+  ``Require'' a module, by Pierre-Marie Pédrot.
+\item Universes: many user-level and API level enhancements: qualified
+  naming and printing, variance annotations for cumulative inductive
+  types, more general constraints and enhancements of the minimization
+  heuristics, interaction with modules by Gaëtan Gilbert, Pierre-Marie
+  Pédrot and Matthieu Sozeau.
+\item Decimal Numbers library by Pierre Letouzey (and DDR?)
+\item Documentation: a large community effort resulted in the migration
+  of the reference manual to the Sphynx documentation tool. The
+  result can be seen at ???
+\item Tools: experimental ``-mangle-names'' option to coqtop/coqc for
+  linting proof scripts, by Jasper Hugunin.
+\end{itemize}
+
+On the implementation side, the \texttt{dev/doc/changes.md} file
+documents the numerous changes to the implementation and improvements of
+interfaces. The file provides guidelines on porting a plugin to the new
+version.
+
+Version 8.8 also comes with a bunch of smaller-scale changes and
+improvements regarding the different components of the system. They are
+documented in the \texttt{CHANGES} file.
+
+The efficiency of the whole system has seen improvements thanks to
+contributions from Gaëtan Gilbert, Pierre-Marie Pédrot, Maxime Dénès and
+Matthieu Sozeau and performance issue tracking by Jason Gross and Paul
+Steckler.
+
+The official wiki and the bugtracker of {\Coq} migrated to the
+github platform, thanks to the work of Pierre
+Letouzey and Théo Zimmermann. Gaëtan Gilbert, Emilio Jesús Gallego Arias
+and ... worked on maintaining and improving the continuous integration
+system.
+
+The OPAM repository for {\Coq} packages has been maintained by Guillaume
+Melquiond, Matthieu Sozeau, Enrico Tassi with contributions from many
+users. A list of packages is available at \url{https://coq.inria.fr/opam/www/}.
+
+TODO Outdated?
```

Comment:
@gares this should be removed, right?

Reply:
Shall I remove the list of packages link? (Can't remember right away how to put an external link in rst either)